Makefile

```
@@ -44,16 +54,19 @@ update_plugin_docs::
 	./scripts/update_plugin_docs.sh azure-native ${PLUGIN_VERSION_AZURE_NATIVE}
 	./scripts/update_plugin_docs.sh awsx ${PLUGIN_VERSION_AWSX}
 
-install::
-	${GO} install ./cmd/...
+.PHONY: install
+install: install_pulumi-language-yaml install_pulumi-converter-yaml
+
+# Install a binary onto GOPATH
+.PHONY: install_%
+install_%: bin/%
+	cp $< $(or $(shell ${GO} env GOBIN),$(shell ${GO} env GOROOT)/bin)/$*
```

```suggestion
	cp $< $(or $(shell ${GO} env GOBIN),$(shell ${GO} env GOPATH)/bin)/$*
```

I think you meant to do this?

---

https://github.com/pulumi/pulumi-yaml/pull/793 because auto-merge was too fast.